<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tcpConnectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpIpLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tcpPortLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tcpDisconnectButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="udpStartListeningButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start listening</string>
    </property>
   </widget>
   <widget class="QPushButton" name="udpStopListeningButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop listening</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="udpPortLineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>port</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="udpTableWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>data 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>data 2</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tcpTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>data 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>data 2</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
